CondFormats/SiPixelTransient/interface/SiPixelTemplate.h

```diff
@@ -682,30 +700,42 @@ class SiPixelTemplate {
   float cota_current_;  //!< current cot alpha
   float cotb_current_;  //!< current cot beta
   float abs_cotb_;      //!< absolute value of cot beta
+  int Dtype_;           //!< flags BPix (=0) or FPix (=1)
```

Comment:
Lower case initial should be used for class members

Reply:
Ok I will change